webapi/src/main/scala/org/knora/webapi/messages/v1/responder/projectmessages/ProjectMessagesV1.scala

```diff
@@ -75,12 +74,18 @@ sealed trait ProjectsResponderRequestV1 extends KnoraRequestV1
 
 // Requests
 /**
-  * Get all information about all projects.
+  * Get all information about all projects. The response is in for of [[ProjectsResponseV1]]
```

Comment:
Their behavior and return type are different. The ``ProjectsGetRequestV1`` returns either something or a ``NotFound`` exception if there are no projects found. The ``ProjectsGetV1`` returns an empty sequence in those cases. Also ``ProjectsGetRequestV1`` will eventually check for the user's permissions for the request, but ``ProjectsGetV1`` will not. The one is used in a route the other internally. We have a few messages like this. They follow this convention:
 - SomethingSomethingRequestV1 -> Used in routes and have always a SomethingSomethingResponseV1 as a response.
 - SomethingSomethingV1 -> Used internally and can have basically anything as a response.